addons/Coloring/src/presenter.js

```diff
@@ -40,7 +40,7 @@ function AddonColoring_create(){
         selected: "Selected",
         correct: "Correct",
         incorrect: "Incorrect",
-        defaultColorName: "Teal"
+        emptyColorName: "Undefined"
```

Comment:
Raczej tego TTS nie będzie, ale zaznaczam gdyby jednak zostało.
To TTS nie jest zdefiniowane w build.xml, słowniczkach ani w upgradeModel.